internal/chainid_test.go

```diff
@@ -2,16 +2,34 @@ package chainid
 
 import "testing"
 
-func TestEncodeChainId(t *testing.T) {
+func TestEncodeMainnetChainId(t *testing.T) {
 	mainnetChainId := EncodeChainId("SN_MAIN")
 	expectedMainnetChainId := "0x534e5f4d41494e"
 	if mainnetChainId != expectedMainnetChainId {
-			t.Errorf("Got %q, expected %q", mainnetChainId, expectedMainnetChainId)
+		t.Errorf("Got %q, expected %q", mainnetChainId, expectedMainnetChainId)
 	}
+}
+
+func TestEncodeGoerliId(t *testing.T) {
+	goerliChainId := EncodeChainId("SN_GOERLI")
+	expectedGoerliChainId := "0x534e5f474f45524c49"
+	if goerliChainId != expectedGoerliChainId {
+		t.Errorf("Got %q, expected %q", goerliChainId, expectedGoerliChainId)
+	}
+}
+
+func BenchmarkEncodeMainnetChainId(t *testing.B) {
+	mainnetChainId := EncodeChainId("SN_MAIN")
+	expectedMainnetChainId := "0x534e5f4d41494e"
+	if mainnetChainId != expectedMainnetChainId {
+		t.Errorf("Got %q, expected %q", mainnetChainId, expectedMainnetChainId)
+	}
+}
```

Comment:
The use of `b` to name `*testing.B` is idiomatic. Also, benchmarks don't need to be tests (i.e checking code behavior or correctness, that's what `TestEncodeMainnetChainId` is for) you only want to "benchmark" the code. To do it correctly, you need to put the function call you want to benchmark in a for loop like the one below, because is the reliable way that Go has to time the function reliably.
```suggestion
func BenchmarkEncodeMainnetChainId(b *testing.B) {
	mainnetChainId := EncodeChainId("SN_MAIN")
	for i := 0; i < b.N; i++ {
            EncodeChainId("SN_MAIN")
        }
}
```